rapids-4-spark-tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -97,4 +99,73 @@ class QualificationSuite extends FunSuite with Logging {
     val logFiles = Array(s"$logDir/nds_q86_test")
     runQualificationTest(logFiles, "nds_q86_test_expectation.csv")
   }
+
+  test("sql metric agg") {
```

Comment:
we could do this separately and if you don't have time this week just let me know, but it would be nice to also validate the sql time aggregation across multiple SQL jobs.  Add in one with dataset for bonus.